app/assets/stylesheets/ama_layout/layout_components/accordions.scss

```diff
@@ -76,6 +76,105 @@
   }
 }
 
+.accordion-multi-state{
+  @extend .accordion;
+  margin-bottom: $base-margin;
+  background: transparent;
+
+  .accordion-item{
+    margin-bottom: $base-margin;
+  }
+
+  &__header{
+    @extend .accordion__header;
+    background: transparent;
+    position: relative;
+  }
+
+  &__heading-bg{
+    @extend .accordion__heading;
+    padding: 0.63rem 1rem !important;
```

Comment:
Hmm I think that we would be better off to just not extend the .accordion__heading, and just redefine the styles. Otherwise, it will be hard to override the !important later if we have to add a modifier. So like:
&__heading-bg{
  @include small-10;
  color: $white;
  padding: 0.63rem $base-padding;
  width: calc(100% - 20px);
}